app/frontend/tests/unit/views/form/Manage.spec.js

```diff
@@ -0,0 +1,74 @@
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import Vuex from 'vuex';
+
+import Manage from '@/views/form/Manage.vue';
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+
+describe('Manage.vue', () => {
+  const mockFormGetter = jest.fn();
+  let store;
+  let formActions = {
```

Comment:
This can be a const - you aren't reassigning the formActions object at any point after the declaration.